### PR TITLE
[CHANGED] Logging when msgs limit is reached

### PR DESCRIPTION
**Pull request by kozlovic**
We now only print the message once per subject.

### CODE REVIEW

File: stores/filestore.go [hunk 1]
```diff
@@ -1112,7 +1112,11 @@ func (ms *FileMsgStore) enforceLimits() error {
 		ms.totalBytes -= firstMsgSize
 
 		// Remove the first message from our cache
-		Noticef("WARNING: Removing message[%d] from the store for [`%s`]", ms.first, ms.subject)
+		if !ms.hitLimit {
+			ms.hitLimit = true
+			Noticef("WARNING: One of the limits (msgs=%v bytes=%v) reached for store [`%s`]",
```
Comment (ColinSullivan1) [inline stores/filestore.go]: WDYT about adding something to the effect that messages will be discarded/unavailable?

Comment (kozlovic) [inline stores/filestore.go]: You're right. I dropped the fact that old messages are going to be removed from the log statement. Should add something to that effect.

Comment (aricart) [inline stores/filestore.go]: How about: "Store limits for ['%s'] store have been exceeded. The store will permanently prune old messages as new messages arrive."

**Comment by ColinSullivan1** [comment]
LGTM.

**Comment by aricart** [comment]
LGTM

**Pull request merged**